### PR TITLE
Use native GRPC port for Node.JS

### PR DESCRIPTION
**Pull request by shumkov**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -10,13 +10,15 @@ class DAPIClient {
...
-   * @param {number} [options.retries] - num of retries if there is no response from DAPI node
+   * @param {number} [options.port=3000] - default port for connection to the DAPI
+   * @param {number} [options.nativeGrpcPort=3010] - Native GRPC port for connection to the DAPI
+   * @param {number} [options.timeout=2000] - timeout for connection to the DAPI
+   * @param {number} [options.retries=3] - num of retries if there is no response from DAPI node
    */
   constructor(options = {}) {
     this.MNDiscovery = new MNDiscovery(options.seeds, options.port);
     this.DAPIPort = options.port || config.Api.port;
+    this.nativeGrpcPort = options.nativeGrpcPort || config.grpc.nativePort;
```
Comment (antouhou) [inline src/index.js]: Why `native` instead of just `port`?

Comment (shumkov) [reply src/index.js]: Because GRPC has two ports, one for GRPC Native one for GRPC Web.

Comment (antouhou) [reply src/index.js]: Where is the web port then?

Comment (shumkov) [reply src/index.js]: For web, we use the same port as JSON RPC.

**Pull request merged**